### ISSUE TITLE
Список команд сделать кликабельным

### ISSUE THREAD

**Issue by ziceptor** [GreWeMa/gwm_Crates#10]
Было бы удобно, что бы нажатие на команду отправляло шаблон в командную строку.

**Comment by GreWeMa** [comment]
Нажатие на команду из /gwmcrates help?

**Comment by iLefty** [comment]
Да, и прочие команды тоже.

**Comment by GreWeMa** [comment]
Ну, дело в том, что это практически нереализуемо (ну либо большими костылями), ибо /gwmcrates help - это кастомно настраиваемый текст из language.conf, и туда вполне можно поставить не команды, а какой угодно текст..

**Comment by ziceptor** [comment]
Не могу сейчас проверить, но в каком-то плагине я видел такое. В программировании нет ничего не возможного, надо просто поставить цель ;)

**Comment by GreWeMa** [comment]
Даже если можно как-то реализовать вставку текста в чат при клике на сообщение в чат,
я не про это,
я про то что сообщение то КАСТОМНОЕ.
Выводиться то - что захочет админ сервера, и там может вообще не быть комманд например, и так далее.

**Comment by iLefty** [comment]
Ты не понял. Смотри, кликаешь на команду, а она автоматом в чат записывается, либо выполняется.

**Comment by ziceptor** [comment]
Да. Вывод в чате можно сделать кликабельным. Если команда без аргументов, то можно сразу выполнять. Если подразумеваются аргументы, то просто вставлять в командную строку.

Я понимаю, под командной строкой, то поле ввода, которое отображается после нажатия /.

Теперь про кастомное сообщение.
Незачем админу давать возможность изменения формата команд. Надо делать локализацию только описания команд. Другими словами - переделать надо.

Я встречал на форуме посты, в которых упоминалось про большое количество ящиков.. вроде больше 40. Мне жаль тех админов! :) А без этого https://github.com/GreWeMa/gwm_Crates/issues/9
втройне жаль ;)

Добавление данного функционала повысит привлекательность плагина, так как юзабилити повысится... станет вразы удобнее пользоваться.

**Comment by ziceptor** [comment]
Напоминалка http://radikal.ru/video/cCumZqnLmqU

**Comment by PixelmonPRO** [comment]
И добавить продолжение команды кнопкой tab

**Comment by GreWeMa** [comment]
Articuno, тут как раз соседняя тема: https://github.com/GreWeMa/gwm_Crates/issues/9
